docs/contributing/backward-compatibility-promise.md

```diff
@@ -22,6 +22,8 @@ Once created, a git tag marking a release will never be removed or edited.
 Pre-release version format is `MAJOR.MINOR.PATCH-<alpha|beta|rc><n>`, eg. `7.0.0-beta5`.*
 
 ### Current Release Plan
+Currently, `7.3`, and `8.0` versions are supported, i.e. we are releasing patch versions for them on a regular basis.
```

Comment:
```suggestion
Currently, `7.3` and `8.0` versions are supported, i.e. we are releasing patch versions for them on a regular basis.
```

There shouldn't be a comma before *and* or *or* if there are only two items.

Reply:
This list should be updated with every `MAJOR` and `MINOR` release, right?

Would you care to create a manual release worker to remind us of that?

Reply:
good idea, added in https://github.com/shopsys/shopsys/pull/1310/commits/499e36105178c1c408dd04721d63f68c5d92fbb6